### ISSUE TITLE
support icon in front of label defining annotation class

### ISSUE THREAD

**Issue by mjy** [SpeciesFileGroup/semanticTextAnnotator#8]
* use data- attributes or css classes

**Comment by morefx** [comment]
What might be a good (example) attribute? E.g. different icons for different ontologies?

**Comment by mjy** [comment]
Yes, perhaps, alternatively perhaps it reflects different annotation template classes?
* annotation describes a shape
* annotation describes a line, point, area
* annotation describes a relative size difference 
... etc?
